tests/components/google_assistant/__init__.py

```diff
@@ -116,7 +116,7 @@
     },
     'traits': ['action.devices.traits.OpenClose'],
     'type':
-    'action.devices.types.BLINDS',
+    'action.devices.types.GARAGE',
```

Comment:
maybe a test with blind and with garage?

Reply:
The test_sync_request() in test_google_assistant.py is not enough? It only changes the device type not how it is handled. What else do I have to test?

Reply:
This is enough. Traits and types are different things.